doc/packages.md

```diff
@@ -195,6 +195,7 @@
 | ooce/storage/minio		| 2024-01-13T07-53-03Z | https://github.com/minio/minio/releases | [omniosorg](https://github.com/omniosorg)
 | ooce/storage/minio-mc		| 2024-01-13T08-44-48Z | https://github.com/minio/mc/releases | [omniosorg](https://github.com/omniosorg)
 | ooce/system/file-system/ntfs-3g | 2022.10.3	| https://github.com/tuxera/ntfs-3g/releases | [omniosorg](https://github.com/omniosorg)
+| ooce/system/gptfdisk		| 1.0.9		| https://sourceforge.net/projects/gptfdisk/files/gptfdisk/ | [omniosorg](https://github.com/omniosorg)
```

Comment:
The last field here is the maintainer, which could be Oxide if that's the intention. No requirement, it can be left as omniosorg too.